src/pkg/cli/loader_test.go

```diff
@@ -16,19 +18,25 @@ func TestLoader(t *testing.T) {
 
 		t.Run(path, func(t *testing.T) {
 			loader := ComposeLoader{path}
-			proj, err := loader.LoadWithProjectName("test")
+			proj, err := loader.LoadCompose(context.Background())
 			if err != nil {
 				t.Fatal(err)
 			}
-			bytes, err := proj.MarshalYAML()
+			yaml, err := proj.MarshalYAML()
 			if err != nil {
 				t.Fatal(err)
 			}
+
+			// Replace the absolute path in context to make the .golden file portable
+			absPath, _ := filepath.Abs(path)
+			yaml = bytes.ReplaceAll(yaml, []byte(filepath.Dir(absPath)), []byte("TESTPATH"))
```

Comment:
instead of `Abs` we got have replaced it with relative path? Would make the golden file more readable, and/or use something more descriptive than `TESTPATH`, like `replaced/for/portability`.

Reply:
Good point, fixed